src/scss/Header.scss

```diff
@@ -0,0 +1,13 @@
+.market_box {
+  display: flex;
+  justify-content: center;
+  background-color: #81B697;
+
+  .market_name {
```

Comment:
css class names should be `kebab-case` :)

Reply:
ok, i've corrected class names according to kebab-case

Reply:
Hej, i've made mistake, because i commited corrections in Tooltip branch :/ 
I didn't improve the filter method to find method, because I'm still reading about it, because when I changed it, unfortunately, there is an error (i.e. nothing renders, i.e. it is immediately false)